bedrock/firefox/templates/firefox/includes/release-menu.html

```diff
@@ -1,11 +1,15 @@
 <nav id="nav-main" role="navigation">
   <span id="nav-main-toggle" class="toggle" role="button" aria-controls="nav-main-menu" aria-expanded="false" tabindex="0">{{_('Menu')}}</span>
   <ul id="nav-main-menu">
-    <li class="first"><a class="more" rel="external" href="https://developer.mozilla.org">{{_('MDN')}}</a>
-    </li>
-    <li><a href="{{ url('firefox') }}">{{_('Firefox')}}</a>
-    </li>
-    <li><a href="{{ url('firefox.developer') }}">{{_('Firefox Developer Edition')}}</a>
-    </li>
+    <li class="first"><a class="more" rel="external" href="https://developer.mozilla.org">{{_('MDN')}}</a></li>
+    <li><a href="{{ url('firefox') }}">{{_('Firefox')}}</a></li>
+    {% if release.product == 'Firefox for Android' %}
+    <li><a href="{{ url('firefox.channel.android') }}">{{_('Firefox Nightly &amp; Beta')}}</a></li>
+    {% elif release.product == 'Firefox for iOS' %}
+    <li><a href="{{ url('firefox.channel.ios') }}">{{_('Firefox Beta')}}</a></li>
+    {% else %}
+    <li><a href="{{ url('firefox.channel.desktop') }}">{{_('Firefox Nightly &amp; Beta')}}</a></li>
```

Comment:
It feels like the menu items are getting in a bit of a muddle having both Nightly and Beta in one link, but Dev Edition separately in another. I get the reason why, since they point to different URLs, but until someone clicks a link it feels a little confusing.

Since all pre-release builds can be found on a single page, another approach could be to cut this down to three simple links:

'MDN', 'Firefox', 'Pre-release builds' (or something similar?)

Thoughts, @kyoshino, @pascalchevrel ?

Also, this old menu will likely be going away sometime in the future, but this will likely happen when this page gets redesigned to fit the new branding.